### PR TITLE
Update Nessie integration doc about Nessie SQL Extensions

### PR DESCRIPTION
**Pull request by ajantha-bhat**
updating the document as per [this](https://github.com/projectnessie/nessie/issues/1979)

### CODE REVIEW

**Comment by ajantha-bhat** [comment]
cc: @nastra , @RussellSpitzer

File: site/docs/nessie.md [hunk 1]
```diff
@@ -33,6 +33,16 @@ The `iceberg-nessie` module is bundled with Spark and Flink runtimes for all ver
 with Nessie and Iceberg simply add the Iceberg runtime to your process. Eg: `spark-sql --packages
 org.apache.iceberg:iceberg-spark3-runtiume:{{ versions.iceberg }}`. 
 
+From spark, Nessie SQL extensions can be used to manage the Nessie repo as shown below.
```
Comment (nastra) [inline site/docs/nessie.md:36]: maybe create a subsection here named `## Spark SQL Extensions`

File: site/docs/nessie.md [hunk 2]
```diff
@@ -33,6 +33,16 @@ The `iceberg-nessie` module is bundled with Spark and Flink runtimes for all ver
 with Nessie and Iceberg simply add the Iceberg runtime to your process. Eg: `spark-sql --packages
 org.apache.iceberg:iceberg-spark3-runtiume:{{ versions.iceberg }}`. 
 
+From spark, Nessie SQL extensions can be used to manage the Nessie repo as shown below. 
+
+```
+bin/spark-sql 
+  --packages "org.apache.iceberg:iceberg-spark3-runtime:{{ versions.iceberg }},org.projectnessie:nessie-spark-extensions:{{ versions.java }}"
```
[the remark this answers is not in the document]
Comment (ajantha-bhat) [reply site/docs/nessie.md]: updated. thanks

File: site/docs/nessie.md [hunk 3]
```diff
@@ -63,6 +73,8 @@ conf.set("spark.sql.catalog.nessie.uri", "http://localhost:19120/api/v1")
 conf.set("spark.sql.catalog.nessie.ref", "main")
 conf.set("spark.sql.catalog.nessie.catalog-impl", "org.apache.iceberg.nessie.NessieCatalog")
 conf.set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog")
+conf.set("spark.sql.catalog.nessie", "org.apache.iceberg.spark.SparkCatalog")
```
Comment (RussellSpitzer) [inline site/docs/nessie.md:76]: Looks like this is duplicated now?

**Review by nastra — APPROVED** [review]
LGTM

**Pull request merged**